jest.config.js

```diff
@@ -6,7 +6,12 @@ module.exports = {
     '.+\\.(css|styl|less|sass|scss)$': `identity-obj-proxy`,
     '.+\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$': `<rootDir>/__mocks__/file-mock.js`,
   },
-  testPathIgnorePatterns: [`node_modules`, `\\.cache`, `<rootDir>.*/public`],
+  testPathIgnorePatterns: [
+    `node_modules`,
+    `\\.cache`,
+    `<rootDir>.*/public`,
+    `cypress`,
```

Comment:
🚒

Reply:
I also ran into needing to add this for CI. should cause a minor merge conflict with master

Reply:
nice, I just cleaned up the conflicts 🚀